packages/charts/src/components/legend/legend.utils.ts

```diff
@@ -88,13 +90,13 @@ export const createSliderTransition = (
     };
   return {
     initial: (direction: number) => ({
-      y: direction > 0 ? itemPositionNext : itemPositionPrevious,
+      y: direction > 0 ? itemPositionNext + 100 : itemPositionPrevious - 100,
```

Comment:
need to discuss later on how to remove these magic numbers

Reply:
No magic there right now, clear math :)